pkgs/build-support/testers/hasPkgConfigModules/tester.nix

```diff
@@ -30,18 +30,19 @@ runCommand testName {
         }
         package.meta;
   } ''
+    touch "$out"
```

Comment:
I don't see an use case for hasPkgConfigModules with `moduleNames = [ ];`.

Reply:
Some generated codes do use this, namely xorg: https://github.com/NixOS/nixpkgs/pull/280122

Reply:
Got in as part of https://github.com/NixOS/nixpkgs/pull/284740

Reply:
I see.

What about

```nix
if moduleNames == [ ]
then lib.warn "hasPkgConfigModules: No modules specified for ${package.name}" pkgs.emptyFile # (or `null`?)
else runCommandNoCC …
```

The passthru stuff shouldn't show up in ofborg and I think it's a bit clearer and avoids useless derivations.

Reply:
Do we really need the warnings?

Reply:
My reasoning is that having this tester enabled implies that there should be some modules *usually*. What do you think @roberth?